src/rules/declaration-block-no-duplicate-properties/README.md

```diff
@@ -29,3 +29,30 @@ a { color: pink; }
 ```css
 a { color: pink; background: orange; }
 ```
+
+## Options
+
+### `ignore: ["consecutive-duplicates"]`
+
+Ignore consecutive duplicated properties.
+
+They can prove to be useful fallbacks for older browsers.
+
+The following patterns are considered warnings:
+
+```css
+p {
+  font-size: 16px;
+  font-weight: 400;
+  font-size: 1rem;
+}
+```
+
+The following patterns are *not* considered warnings:
+
+```css
+p {
+  font-size: 16px;
+  font-size: 1rem;
```

Comment:
Can you please add `font-weight: 400;` below the `font-size`s, so that it matches the previous sample?